rollup.config.js

```diff
@@ -66,7 +66,9 @@ export default {
       // Don't emit CSS - it doesn't work properly with Web Components.
       emitCss: false,
       compilerOptions: {
-        customElement: false
+        customElement: false,
+        css: 'injected',
+        accessors: true,
```

Comment:
There's a warning that `accessors: true` isn't supported in runes mode, which seems odd